.github/workflows/deploy.yml

```diff
@@ -20,10 +20,14 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
-    - name: Clone apisix's docs from repo
+    - name: Clone apisix's docs from repo & update image's link
       run: |
         git clone https://github.com/apache/apisix.git
+        cp -r apisix/doc/images ./website/static && rm -rf apisix/doc/images
```

Comment:
We must copy all static images under the `./website/static` folder according to docusaurus